src/python/foglamp/core/server.py

```diff
@@ -17,13 +20,26 @@
 
 def _make_app():
     """create the server"""
+    # https://aiohttp.readthedocs.io/en/stable/_modules/aiohttp/web.html#run_app
     app = web.Application(middlewares=[middleware.error_middleware])
     routes.setup(app)
     return app
 
 
+def _shutdown(loop):
+    scheduler.shutdown()
+    for task in asyncio.Task.all_tasks():
+        task.cancel()
+    loop.stop()
+
+
 def start():
     """starts the server"""
-    # https://aiohttp.readthedocs.io/en/stable/_modules/aiohttp/web.html#run_app
+    loop = asyncio.get_event_loop()
+    scheduler.start(loop)
```

Comment:
remove `loop` arg.

**note:** when process is running in background, `foglamp start`. `lsof -i:5683` lists 1 pid on UDP port 5683.
we can still run `python -m foglamp.device` and it will also start scheduler at port 5683 in foreground  with new pid (i.e. `lsof -i:5683` lists 2 pid)

> For UDP (Multicasts), multiple applications can subscribe to the same port.